eng/pipelines/aggregate-reports.yml

```diff
@@ -128,3 +94,38 @@ jobs:
       condition: succeededOrFailed()
       displayName: 'Publish Report Artifacts'
       artifact: reports
+  - job: Compliance
+    timeoutInMinutes: 120
+    variables:
+      - template: templates/variables/globals.yml
+    pool:
+      vmImage: 'windows-2019'
+    steps:
+    - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
+      # ComponentGovernance is currently unable to run on pull requests of public projects. Running on non-PR
```

Comment:
Based on discussion, we can move the link check step here as it does not need repository to be built